python/ray/data/tests/test_huggingface.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import ray
+import datasets
```

Comment:
We'll need to add this as a test dependency here: https://github.com/ray-project/ray/blob/master/python/requirements/data_processing/requirements_dataset.txt

Reply:
`python/requirements/data_processing/requirements_dataset.txt` or `python/requirements/data_processing/requirements.txt`?

Reply:
The former:
- https://github.com/ray-project/ray/blob/09f1cf26dc590dc3539938a3e6fc41703e527f27/.buildkite/pipeline.ml.yml#L352
- https://github.com/ray-project/ray/blob/09f1cf26dc590dc3539938a3e6fc41703e527f27/ci/env/install-dependencies.sh#L389-L391